docs/getting_started.Rmd

```diff
@@ -135,38 +132,12 @@ board.pin_meta("mtcars")
 While we’ll do our best to keep the automatically generated metadata consistent over time, I’d recommend manually capturing anything you really care about in metadata.
 
 
-## 🚧 Versioning
-
-
-> ⚠️: Warning the examples in this section use joblib to read and write data. Joblib uses the pickle format, and **pickle files are not secure**. Only read pickle files you trust. In order to read pickle files, set the `allow_pickle_read=True` argument. See: https://docs.python.org/3/library/pickle.html.
-
-
-> ⚠️: Turning off versioning is not yet implemented; all Python pins are versioned. These docs are copied from R pins.
-
-In many situations it's useful to version pins, so that writing to an existing pin does not replace the existing data, but instead adds a new copy.
-There are two ways to turn versioning on:
-
--   When you create a board you can turn versioning on for every pin in that board:
-
-    ```python
-    board2 = board_temp(versioned = TRUE)
-    ```
-
--   When you write a pin, you can specifically request that versioning be turned on for that pin:
-
-    ```python
-    board2 = board_temp()
-    board2.pin_write(mtcars, versioned = TRUE)
-    ```
+## Versioning
 
-Most boards have versioning on by default.
-The primary exception is `board_folder()` since that stores data on your computer, and there's no automated way to clean up the data your saving.
-
-Once you have turned versioning on, every `pin_write()` will create a new version:
+Every `pin_write()` will create a new version:
 
 ```{python}
 board2 = board_temp(versioned = True, allow_pickle_read=True)
-
 board2.pin_write([1,2,3,4,5], name = "x", type = "joblib", title="TODO")
 board2.pin_write([1,2,3], name = "x", type = "joblib", title="TODO")
 board2.pin_write([1,2], name = "x", type = "joblib", title="TODO")
```

Comment:
```suggestion
board2.pin_write([1,2,3,4,5], name = "x", type = "joblib")
board2.pin_write([1,2,3], name = "x", type = "joblib")
board2.pin_write([1,2], name = "x", type = "joblib")
```

Is that an OK change? 

Do you think it looks really weird to store this as binary? Should we change to JSON?

Reply:
Ah, changed to JSON, and added a new section on storing models, WDYT:

<img width="1238" alt="image" src="https://user-images.githubusercontent.com/2574498/200873258-32ecb19d-180e-4e49-9c5a-3620d7f23ba9.png">

Reply:
This looks great! 🙌